manifests/config.pp

```diff
@@ -31,8 +31,14 @@
   }
 
   file { $::foreman::init_config:
-    ensure  => file,
-    content => template("foreman/${foreman::init_config_tmpl}.erb"),
+    ensure  => absent,
+  }
+
+  if $::foreman::use_foreman_service {
+    systemd::dropin_file { 'installer.conf':
```

Comment:
Why is it called `installer.conf` ?

Reply:
I thought it'd make it easy to track where it came from. It ends up as `/etc/systemd/system/foreman.service.d/installer.conf`